modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -1615,6 +1622,20 @@ else if (status == WorkflowConstants.STATUS_APPROVED) {
 		return dynamicQuery.add(junction);
 	}
 
+	private void _checkKBArticlesByDisplayDate(long companyId, Date displayDate)
+		throws PortalException {
+
+		if (_log.isDebugEnabled()) {
+			_log.debug(
+				StringBundler.concat(
+					"Publishing file entries with display date previous to ",
+					displayDate, " for company ", companyId));
+		}
+
+		_publishKBArticlesByCompany(
+			_companyLocalService.getCompany(companyId), displayDate);
```

Comment:
Would it be possible to inline this method? Otherwise, `_checkKBArticlesByDisplayDate` does nothing but log and delegate, which feels odd.

Reply:
Hi, @adolfopa 

I thought about doing it, but later I see that the structure that `_checkKBArticlesByExpirationDate` and `_checkKBArticlesByReviewDate` is the same (the log and then the `_expireKBArticlesByCompany` /`_notifyReviewKBArticlesByCompany`) is the same as now.

What are your thoughts about it?

let them as is
inline all?
inline only the new?